### DISCUSSION TITLE
HOW DO I PREVENT FINGERPRINTING?

### DISCUSSION THREAD

**Discussion by Skorchekd** [ZFC-Digital/puppeteer-real-browser#222] [General]
a site can detect the browser via the unique fingerprint. i wish to find out how to prevent this. any help is appreciated. i do n0ot wish to use the puppeteer-with-fingerprints library as it is detected and cannot be implemented with this library. is there any effective solution? thanks.

**Comment by RubberArchind** [comment]
your browser detected in https://bot-detector.rebrowser.net/ again?
